sdks/java/nexmark/src/test/java/org/apache/beam/sdk/nexmark/queries/QueryTest.java

```diff
@@ -205,12 +206,14 @@ public void sqlQuery5MatchesModelStreaming() {
         new Query5Model(CONFIG), true);
   }
 
+  @Ignore("TODO: BEAM-3816")
```

Comment:
Just for future lazy people, can you just put the whole URL like `@Ignore("https://issues.apache.org/jira/browse/BEAM-3816")`

Reply:
Done.